app/views/katello/api/v2/host_collections/_host_collection.json.rabl

```diff
@@ -1,4 +1,5 @@
 attribute :pulp_id, :name, :organization_id, :max_hosts, :description, :total_hosts, :unlimited_hosts
+attributes :host_ids
```

Comment:
@bbuckingham he's saying to move it to `show.json.rabl` instead of `_host_collection.json.rabl`.  Does the bug expect `host_ids` to exist in the index response?